bin/GameIndex.yaml

```diff
@@ -5082,6 +5108,8 @@ SCUS-97381:
   region: "NTSC-U"
   gameFixes:
     - VU0KickstartHack # Fixes Character SPS.
+    - EETimingHack # Fixes DMA errors 
```

Comment:
Dito :)

Reply:
Have you played through the US version to make sure it isn't required?